bfdd/bfdd_nb_config.c

```diff
@@ -668,10 +864,13 @@ int bfdd_bfd_sessions_single_hop_desired_transmission_interval_modify(
 int bfdd_bfd_sessions_single_hop_required_receive_interval_modify(
 	struct nb_cb_modify_args *args)
 {
+	uint32_t rx_interval = yang_dnode_get_uint32(args->dnode, NULL);
 	struct bfd_session *bs;
 
 	switch (args->event) {
 	case NB_EV_VALIDATE:
+		if (rx_interval < 10000 || rx_interval > 60000000)
+			return NB_ERR_VALIDATION;
 		break;
```

Comment:
ditto: yang change
separate commit for changes related to common bfd.

Reply:
you seem to have removed those changes.
if the behavior can be kept unchanged, it is ok to me.